cmd/anonymizer/app/anonymizer/anonymizer.go

```diff
@@ -53,11 +53,11 @@ type mapping struct {
 //
 // The mapping from original to obfuscated strings is stored in a file and can be reused between runs.
 type Anonymizer struct {
-	mappingFile       string
-	logger            *zap.Logger
-	lock              sync.Mutex
-	mapping           mapping
-	options           *Options
+	mappingFile string
+	logger      *zap.Logger
+	lock        sync.Mutex
+	mapping     mapping
+	options     *Options
```

Comment:
nit: you can assign Options by value. Doesn't matter much in this case, but if this was somewhere on a hot path of high throughput service, then a pointer is unnecessary indirection.